ci/changelog-configuration.json

```diff
@@ -2,15 +2,15 @@
     "pr_template": "- ${{TITLE}} (#${{NUMBER}})",
     "categories": [
```

Comment:
Perhaps we could add or replace the existing labels here too;
`type: feature`
`info: UI`
`type: documentation`?

Reply:
Afaik the labels on github are shared between Issues and PRs, I'm not sure how or what for those `type: ` and `info: ` labels are used :man_shrugging: 
My guess is that they were used in an effort to categorize open issues some time ago. But no objection from my side to consolidate or clean up.

Also the labels/categories I used here for the PR template / changelog are totally up for debate. It's really just a first suggestion of what I thought could be useful, but not hard preference whatsoever.

Reply:
@haasad @marc-vdm
First of all I don't know too much about the standards when it comes to github actions. This is my first exposure to them, the other projects I have been involved with have been very much for internal use only. So apologies if I run roughshod over some standard conventions/requirements, I will try not to be rude.

I generally like this and really only have a single issue, which is with the label `Changes`, to me this is like a catch-all and too broad a label to be relevant to us in reviewing a pull request.

Reply:
@haasad FYI: @Zoophobus is the new AB developer who recently started with us. We'll be hearing more from him :)